src/pages/Pools/PairsByToken0/index.tsx

```diff
@@ -0,0 +1,201 @@
+                    <Text fontWeight="600" fontSize="16px" lineHeight="20px">
+                      {token0?.symbol}
+                    </Text>
+                  </Box>
+                </PointableFlex>
+                <Box>
+                  <ChevronDown size={12} />
+                </Box>
+              </Flex>
+              <ButtonRow>
+                <ResponsiveButtonPrimary id="join-pool-button" as={Link} padding="8px 14px" to="/create">
+                  <Text fontWeight={700} fontSize={12}>
+                    CREATE PAIR
+                  </Text>
+                </ResponsiveButtonPrimary>
+                {liquidityMiningEnabled && (
+                  <ResponsiveButtonSecondary as={Link} padding="8px 14px" to="/liquidity-mining/create">
+                    <Text fontWeight={700} fontSize={12} lineHeight="15px">
+                      CREATE LIQ. MINING
+                    </Text>
+                  </ResponsiveButtonSecondary>
+                )}
+              </ButtonRow>
+            </TitleRow>
+            <PairsList loading={loadingPairs} pairs={pairs} />
+          </AutoColumn>
+        </AutoColumn>
+        {/* TODO: uncomment once multi-chain analytics are a thing */}
+        {/* <ButtonWithLink
+          link={`https://dxstats.eth.link/#/account/${account}`}
+          text={'ACCOUNT ANALYTICS AND ACCRUED FEES'}
+          marginTop="32px"
+        /> */}
+        {/* Should not be needed since when we fetch liquidity positions from the subgraph */}
+        {/* <TYPE.body color="text4" textAlign="center" fontWeight="500" fontSize="14px" lineHeight="17px" marginTop="32px">
+          Don't see a pool you joined?{' '}
+          <StyledInternalLink color="text5" id="import-pool-link" to="/find">
+            Import it.
+          </StyledInternalLink>
+        </TYPE.body> */}
+
+        <VoteCard style={{ marginTop: '32px' }}>
+          <CardSection>
+            <AutoColumn gap="md">
+              <RowBetween>
+                <TYPE.body fontWeight={600} lineHeight="20px">
+                  Liquidity provider rewards
+                </TYPE.body>
+              </RowBetween>
+              <RowBetween>
+                <TYPE.body fontWeight="500" fontSize="12px" lineHeight="20px" letterSpacing="-0.4px">
+                  Liquidity providers earn a swap fee (0.25% by default) on all trades proportional to their share of
+                  the pool.
+                  <br /> Fees are added to the pool, accrue in real time and can be claimed by withdrawing your
+                  liquidity.
+                  <br /> The swap fee value is decided by DXdao and liquidty providers, it can be between 0% and 10% and
+                  it uses 0.25% as default value that is assigned when the pair is created.
+                </TYPE.body>
+              </RowBetween>
+              {/*<RowBetween>*/}
+              {/*  /!* TODO: this should be a link to a blog post or something *!/*/}
```

Comment:
Do we have this yet?  If not should make it a TODO for marketing / comms

Reply:
We definitely don't have it, would be nice to, so a todo makes sense